pkgs/misc/vim-plugins/default.nix

```diff
@@ -1565,4 +1565,36 @@ rec {
 
   };
 
+  vim-github-colorscheme = buildVimPluginFrom2Nix { # created by nix#NixDerivation
+    name = "vim-colorscheme-2015-03-09";
```

Comment:
No, I reinserted `-github-` because it is part of the repository name: *mmazer/vim-github-colorscheme*. Calling this package `vim-colorscheme` wouldn't make sense.